packages/components/ng-ovh-payment-method/src/components/choice/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,33 @@
+{
+  "ovh_payment_method_choice_title": "Afin de confirmer votre demande, veuillez choisir votre moyen de paiement, et accepter les conditions d'utilisation.",
+  "ovh_payment_method_choice_tag_recommended": "Conseillé",
+  "ovh_payment_method_choice_payment_method_add_link": "[Rendez-vous sur la page <a href=\"#/billing/mean\">Moyens de paiement</a> pour en sélectionner un]",
```

Comment:
What's happen if we are not in `dedicated` manager ?

Reply:
https://github.com/ovh/manager/pull/1871#discussion_r352131476 ?

Reply:
Yes, as it a separate package we can't ensure that `#/billing/mean` is implemented in the host application.

Reply:
This means, that we have to construct the whole manager url to redirect him to the valid page, but this url is built on user's language, How can we get this value (`ovhSubsidiary`) as he may not have this setting ?
Or maybe we hide it if we can't build the url.

Reply:
Maybe you could use a provider that sets the url at the app start ? That way we on manager side can configure it in @ovh-ux/manager-core for instance but it can be used in some other apps.

Reply:
Yes I wrote `language` but was meaning the sub, my bad.
Ok with the provider, it seems obvious in fact.